org.eclipse.winery.edmm/src/main/java/org/eclipse/winery/edmm/utils/ZipUtility.java

```diff
@@ -0,0 +1,88 @@
+package org.eclipse.winery.edmm.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+public class ZipUtility {
```

Comment:
There's overlap here to the Zip methods exposed by `org.eclipse.winery.yaml.common.Utils` that allow zipping and unzipping as well as the Zipping and unzipping in FilebasedRepository and quite probably other places in the codebase...

It's probably worth checking for a way to unify these, because zipping can be non-obvious.

Reply:
Yes, you are absolutely right. I can't work on this right now, but I'll check in the next days if this class can be removed or unified with the other zipping functions around the code. 

The idea is to just use the `org.eclipse.winery.yaml.common.Utils` zip function and remove the other copies, is this right?

Reply:
I don't have an exhaustive overview where in winery Zip handing is done, but I assume there are multiple different usage patterns that are not that easy to unify. The idea is to centralize the Zip handling into a single utility class. That class probably *should* be in `org.eclipse.winery.common`.

That change could be a bit out-of-scope for this PR, though... I'm not sure on the constraints that are at play here.

Reply:
Ok, I will check if I can get rid of this "duplicated" code using one zip utility already implemented.
If it's not possible I'll apply the changes suggested below, but I won't unify the zip functions since it's not the scope of this PR. 
It's anyway better not to mark this conversation as resolved because this zip utility problem needs to be fixed in the right way.